_news/2019-09-17-deprecate-python-installer.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Terraform: Official Installation Path of Forseti Security
+author: Hannah Shin
+---
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based 
+upgrade path was slow and inefficient, so we're focusing on delivering a 
+faster, simpler, and more consistent experience with Terraform. 
+
+Starting with Forseti Security v2.22.0, Forseti will support Terraform as the 
+official installation path. The Python based Deployment Manager installer will 
+be deprecated effective October 3, 2019, and will no longer be available as an 
+installation option for future releases. 
+
+We have created a migration script and documentation to help you seamlessly 
+migrate from Deployment Manager to Terraform:
+
+* For v2.18.0+ users, refer to the instructions to migrate 
+[here]({% link _docs/latest/setup/migrate.md %}).
+
+* For versions earlier than v2.18.0, please upgrade to v2.18.0 through 
```

Comment:
What would be the amount of work to have to migration script cover to v2.13 (last 10 versions)?